pkgs/applications/networking/localsend/default.nix

```diff
@@ -2,12 +2,12 @@
 
 let
   pname = "localsend";
-  version = "1.11.0";
+  version = "1.11.1";
 
   srcs = rec {
     x86_64-linux = fetchurl {
       url = "https://github.com/localsend/localsend/releases/download/v${version}/LocalSend-${version}-linux-x86-64.AppImage";
-      hash = "sha256-IIxknLzlAjH27o54R0Zm7T8bhDRAIgh58Evs7zPMrPk=";
+      hash = "sha256-K4M9cks0FNsCLIqQhSgUAz3tRMKng6JkZ/ZfwG2hZJA=";
```

Comment:
I guess hash for darwin should be updated as well.

Reply:
Now there's an update script that will handle both.